tap/extensions/amqp/go.mod

```diff
@@ -1,7 +1,9 @@
 module github.com/up9inc/mizu/tap/extensions/amqp
 
-go 1.16
+go 1.17
 
 require github.com/up9inc/mizu/tap/api v0.0.0
 
+require github.com/google/martian v2.1.0+incompatible // indirect
```

Comment:
why does the AMQP has reference to martian?

Reply:
I guess it is because the "tap/api" module has maritan?

Reply:
yep